src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/resample/resample_kernel_opt.cpp

```diff
@@ -116,17 +139,19 @@ bool ResampleKernelOpt::Validate(const Params& p, const optional_params& o) cons
         return false;
 
     const auto& input = params.inputs[0];
+    const auto & output = params.outputs[0];
 
     if ((input.GetDType() == Datatype::UINT8 || input.GetDType() == Datatype::INT8) &&
         params.resampleType != ResampleType::NEAREST_NEIGHBOR &&
-        params.resampleType != ResampleType::BILINEAR_INTERP)
+        params.resampleType != ResampleType::BILINEAR_INTERP &&
+        params.resampleType != ResampleType::LINEAR_ONNX)
         return false;
 
-    if (input.GetLayout() != DataLayout::fs_b_yx_fsv32 &&
-        input.GetLayout() != DataLayout::b_fs_yx_fsv16 &&
-        input.GetLayout() != DataLayout::b_fs_yx_fsv32 &&
-        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv16 &&
-        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv32)
+    if (params.resampleType == ResampleType::LINEAR_ONNX &&
+        input.Dimentions() == 5 &&
+        (input.Batch().v != output.Batch().v ||
+        input.Feature().v != output.Feature().v ||
+        input.Z().v != output.Z().v))
         return false;
 
```

Comment:
This PR only supports nearest resample type for 5D. Right? In the case of the opt kernel, 5D tensors were not previously supported. It seems that the Validate() function for other resample types needs to return false if input/output tensor is 5D tensor.

Reply:
Added checking of nearest resample for this checking